rest/rest-cli/src/test/java/org/ow2/proactive_grid_cloud_portal/cli/CommandSetTest.java

```diff
@@ -38,86 +40,98 @@
 
 import com.google.common.collect.ImmutableList;
 
-
 /**
- * The class is in charge to test that all {@link org.ow2.proactive_grid_cloud_portal.cli.CommandSet.Entry}
- * defined in {@link CommandSet} class have their number of expected parameters that match the number of
- * tokens used the argNames.
+ * The class is in charge to test that all
+ * {@link org.ow2.proactive_grid_cloud_portal.cli.CommandSet.Entry} defined in
+ * {@link CommandSet} class have their number of expected parameters that match
+ * the number of tokens used the argNames.
  */
 @RunWith(Parameterized.class)
 public class CommandSetTest {
 
-    private CommandSet.Entry entry;
+	private CommandSet.Entry entry;
+
+	// field used by reflection with Junit
+	private String testName;
+
+	public CommandSetTest(CommandSet.Entry entry, String testName) {
+		this.entry = entry;
+		this.testName = testName;
+	}
+
+	@Test
+	public void testThatArgNamesMatchNumberOfArgs() throws ParseException, IllegalAccessException {
+		int nbArgsBasedOnName = Option.UNINITIALIZED;
+		String regex = "\\[.*\\]";
+		Pattern pattern = Pattern.compile(regex);
+		String optionalArgNames = "";
+		int optionals = 0;
+
+		if (entry.argNames() != null) {
+			String argNames = entry.argNames();
 
```

Comment:
This file is reformed, it's hard to see where is changed. By the way, for the format of code, we need to apply the ActiveEon Coding Rule: https://github.com/ow2-proactive/coding-rules if not the build will failed.